cityofaustin/atd-vz-data
# Timezone bug in VZE fatalities view and fix engineering area label

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/18930
Closes https://github.com/cityofaustin/atd-data-tech/issues/18955

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch, start your local db, apply up migrations and metadata. 
2. Go to the Fatalities page in the VZE, filter the dates to be between December 2019 and January 2020, sort by the crash date column.
3. See that the engineering area column now shoes the actual label of the engineering area and not the id
4. See that this [crash](https://visionzero.austin.gov/editor/#/crashes/17492001) (cris_crash_id: 17492001) which occurred on the last day of 2019 is no longer is showing that its the 1st fatality of the year in the ytd_fatality column, it should be 88.
5. Test running the down migrations


---
#### Ship list
